test/Token.js

```diff
@@ -18,33 +23,26 @@ describe("Token contract", function () {
   // lifecycle. These are: `before`, `beforeEach`, `after`, `afterEach`.
 
   // They're very useful to setup the environment for tests, and to clean it
-  // up after they run.
-
-  // A common pattern is to declare some variables, and assign them in the
-  // `before` and `beforeEach` callbacks.
-
-  let Token;
-  let hardhatToken;
-  let owner;
-  let addr1;
-  let addr2;
-  let addrs;
+  // up after they run. However, we use a Hardhat-specific pattern here:
 
-  // `beforeEach` will run before each test, re-deploying the contract every
-  // time. It receives a callback, which can be async.
-  beforeEach(async function () {
+  // The recommended pattern is to provide a "fixture" function, to be used as
+  // an argument to `loadFixture`, which will call it to initialize the chain
+  // state and then return an object with any information that tests will need.
+  async function deployToken() {
```

Comment:
Let's call it `deployTokenFixture`. Using the suffix `Fixture` makes it clear that the fixture is the function itself.

Reply:
Addressed in https://github.com/NomicFoundation/hardhat-boilerplate/pull/61/commits/26f3ec35cd5ecb101c23b95820fadd4a29cebf00